src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const perDay = 40;
+  let total = days * perDay;
+
+  if (days >= 7) {
+    total = total - 50;
+  } else if (days >= 3) {
+    total = total - 20;
```

Comment:
Replace the hard-coded value '3' with a named constant to clearly define what this magic number represents. Also, replace '20' with a named constant to indicate the discount amount for clarity and maintainability.